fluent-gecko/makefile

```diff
@@ -7,52 +7,20 @@ version = $(1)@$(shell node -e "\
 	console.log(require('../$(1)/package.json').version)")
 
 test:
```

Comment:
Should this be an empty recipe? https://www.gnu.org/software/make/manual/html_node/Empty-Recipes.html

Reply:
Ah, thanks, I forgot about this syntax.

Reply:
I also have a Makefule cleanup in store, but that's for later :)